Server.MirForms/Account/PlayerInfoForm.cs

```diff
@@ -29,41 +29,55 @@ public PlayerInfoForm(uint playerId)
             UpdateTabs();
         }
 
+        #region PlayerInfo
         private void UpdatePlayerInfo()
         {
-            IndexTextBox.Text = Character.Index.ToString();
+
+            if (Character == null) return;
+            if (Character.Player == null) return;
+
+                IndexTextBox.Text = Character.Index.ToString();
             NameTextBox.Text = Character.Name;
             LevelTextBox.Text = Character.Level.ToString();
-            ExpTextBox.Text = $"{string.Format("{0:#0.##%}", Character.Player.Experience / (double)Character.Player.MaxExperience)}";
             PKPointsTextBox.Text = Character.PKPoints.ToString();
             GoldTextBox.Text = $"{Character.AccountInfo.Gold:n0}";
             GameGoldTextBox.Text = String.Format("{0:n0}", Character.AccountInfo.Credit);
 
-            ACBox.Text = $"{Character.Player.Stats[Stat.MinAC]}-{Character.Player.Stats[Stat.MaxAC]}";
-            AMCBox.Text = $"{Character.Player.Stats[Stat.MinMAC]}-{Character.Player.Stats[Stat.MaxMAC]}";
-            DCBox.Text = $"{Character.Player.Stats[Stat.MinDC]}-{Character.Player.Stats[Stat.MaxDC]}";
-            MCBox.Text = $"{Character.Player.Stats[Stat.MinMC]}-{Character.Player.Stats[Stat.MaxMC]}";
-            SCBox.Text = $"{Character.Player.Stats[Stat.MinSC]}-{Character.Player.Stats[Stat.MaxSC]}";
-            ACCBox.Text = $"{Character.Player.Stats[Stat.Accuracy]}";
-            AGILBox.Text = $"{Character.Player.Stats[Stat.Agility]}";
-            ATKSPDBox.Text = $"{Character.Player.Stats[Stat.AttackSpeed]}";
 
             if (Character.Player != null)
```

Comment:
double check here - you've already done a player check earlier, no need to do it here